### PR TITLE
Added instructions on how to add chart repository

### PR DESCRIPTION
**Pull request by JGrzybowski**
It's a small PR with something that might be handy to all the people that come from non-containerized world.

When I first discovered this repo it was hard for me to find where to find the URL of the chart repo. 
It would be great to be able to find it with search.

I accept the repositories CLA.

### CODE REVIEW

File: charts/ibm-mq/README.md [hunk 1]
```diff
@@ -43,6 +43,12 @@ See the **configuration** section for how to configure these values.
 
 > **Tip**: A [samples directory](../../samples/README.md) is available that demonstrates how to install the chart in a number of common environments. For first time users it is suggested that this is used.
 
+Add reposity to helm with the command:
```
Comment (callumpjackson) [inline charts/ibm-mq/README.md:46]: Could we change this to:
"To add the helm chart to your helm repository, run the command:"

**Comment by JGrzybowski** [comment]
Also, please add `hacktoberfest` or `hacktoberfest-accepted` tag if this will get merged.

File: charts/ibm-mq/README.md (hunk 1 above)
Comment (JGrzybowski) [reply charts/ibm-mq/README.md]: Changed

**Comment by JGrzybowski** [comment]
@callumpjackson can you add `hacktoberfest-accepted` label to this PR? I would much appreciate it.
It keeps my motivation high to do more for Open Source projects.

**Comment by callumpjackson** [comment]
done

**Pull request merged**